shiny/ui/_chat.py

```diff
@@ -489,7 +489,12 @@ def messages(
 
         return tuple(res)
 
-    async def append_message(self, message: Any) -> None:
+    async def append_message(
+        self,
+        message: Any,
+        *,
+        assistant_icon: HTML | Tag | None = None,
```

Comment:
I think this should be `icon` or `icon_assistant`. `icon_assistant` is consistent with the `chat_ui()` parameter name. But I think here `icon` is okay and we'll document that it's currently only used for the assistant message. It's easy to imagine a future where we include icons for the user message or for other types of messages, in which case it'd be easier to just have an `icon` parameter.

Reply:
I made this change in 12d6de3